tests/integration/functions/runtime_context/string_parameters_test.ts

```diff
@@ -0,0 +1,156 @@
+import {
+  assert, IsExact
+} from "../../../../src/dev_deps.ts";
+import { CanBe, CanBeUndefined } from "../../../../src/test_utils.ts";
+import { DefineFunction } from "../../../../src/mod.ts";
+import {
+  EnrichedSlackFunctionHandler,
+} from "../../../../src/functions/types.ts";
+import { SlackFunctionTester } from "../../../../src/functions/tester/mod.ts";
+import { assertEqualsTypedValues } from "../../../../src/test_utils.ts";
+
+/**
+ * Custom Function handler tests, exercising string-based inputs and outputs
+ */
+Deno.test("Custom Function with a string input and no outputs should provide the string input in a function handler context", () => {
+  const TestFn = DefineFunction({
+    callback_id: "test",
+    title: "test fn",
+    source_file: "test.ts",
+    input_parameters: {
+      properties: {
+        in: {
+          type: "string",
+        },
+      },
+      required: ["in"],
+    },
+  });
+  const handler: EnrichedSlackFunctionHandler<typeof TestFn.definition> = (
+    { inputs },
+  ) => {
+    assert<IsExact<typeof inputs.in, string>>(true);
+    return {
+      outputs: {},
+    };
+  };
+  const { createContext } = SlackFunctionTester(TestFn);
+  const inputs = { in: "test" };
+  const result = handler(createContext({ inputs }));
+  assertEqualsTypedValues(result.outputs, {});
+});
+
+Deno.test("Custom Function with a string input and output should provide the string input in a function handler context", () => {
```

Comment:
Just noticing the name for this test mentions the string input in context, but doesn't really touch on the output which is what distinguishes this test from the one above. Should this test even care about the `input` if that's covered by the above test?

Reply:
Good call. Taking a closer look, maybe we should have dedicated tests for the inputs which assert that within a runtime context, the input is typed properly. For the outputs, we would assert that not providing the correct output name and type raises an error.
WDYT?

Reply:
Yeah, this feels inline with our [other thread here](https://github.com/slackapi/deno-slack-sdk/pull/134#discussion_r1097784986)

Reply:
Done in 09eca8a